notify/impl.go

```diff
@@ -577,6 +577,10 @@ func (n *PagerDuty) notifyV2(
 		n.conf.Severity = "error"
 	}
 
+	summary := tmpl(n.conf.Description)
+	if len(summary) > 1024 {
```

Comment:
This won't be always correct if the string contains UTF-8 characters larger than a byte.

https://play.golang.org/p/LSIA-RjSXHU

Reply:
Fixed via: https://play.golang.org/p/SbFfy6LLUiZ